standards/naming-domains.md

```diff
@@ -4,25 +4,33 @@ category: Operating services
 # Naming domains
 
 All services with real users and which look like GOV.UK should be on
-`*.service.gov.uk` or `*.service.justice.gov.uk`.
+`*.service.gov.uk` or `*.service.justice.gov.uk`. MOJ infrastructure or
+justice-wide services may be eligible for a `*.justice.gov.uk` domain.
 
-### service.gov.uk
+### `service.gov.uk`
 
 The [service standard](https://www.gov.uk/service-manual/technology/get-a-domain-name)
 already sets out when and how to get a `*.service.gov.uk` subdomain:
 public-facing services which have passed their beta assessment should
 be able to get one.
 
-### service.justice.gov.uk
+### `service.justice.gov.uk`
 
 All other services with real users and which look like GOV.UK should be
 on here, including:
 
 - public-facing services which haven’t yet passed their beta assessment
 (but may be in private beta with a restricted set of real users)
-- internal services which look like GOV.UK (although [they can’t use the
+- internal services, which may look like GOV.UK (although [they can’t use the
 crown, New Transport font etc](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk))
 
+### `justice.gov.uk`
+
+Only services which are core, MOJ-wide infrastructure, like email or
+videoconferencing, or otherwise applicable to users across the entire
+justice system (like the intranet) may use a `*.justice.gov.uk`
```

Comment:
@jennyd Apologies if this is explained somewhere already, but I'm not clear about why we specify `service.` on the justice domain. If we can drop it in some circumstances, maybe we can drop it in all? Speaking from the point of view as someone with a long service name already.

Reply:
Hey @davidread, there are a couple of reasons I can recall right now:

- Mirroring GOV.UK, in an attempt to imply consistency between standards of what should be on there.
- We wanted a new, clean space to put new services under. That way, we can have a short list of things that we *want* under `justice.gov.uk`, and gradually chip away at cleaning up the rest (of which there are something like 700 domains, with poor consideration of standards).

I think we had more reasons when we first introduced it, but I honestly forget what they are right now, and I'm not sure we wrote them down 🤦‍♂️

Reply:
Makes sense - thanks. Maybe useful to include this in the text sometime.

Reply:
I think this touches on an assumption I was making to start with on this: that we're mainly concerned with domains for services. Is that true? We also have various static websites (some of which aren't on `*.gov.uk`), and the example of email is a good reminder that MOJ Digital & Technology isn't only about web-based services.

Is guidance that we put in this repo aiming to cover every area that D&T works in? If so we should include a wider range of people in writing and reviewing guidance, or if not we should be more explicit about its limits.

Reply:
On that basis, long service names were not envisaged to be a concern, because people would "Start on GOV.UK". (I'm not sure how well that played out).

Do we have a similar route into things on `*.service.justice.gov.uk`?

Reply:
@jennyd Yes, this guidance is intended to cover *all* of MOJ D&T, and all the domains that implies. It should probably include a section about “other domains” or something to cover the non-`*.gov.uk` domains and those that we host/tend to that are on another GOV.UK subdomain.

@jabley Ideally, the MOJ Intranet should be the place that people start, I think. Except when it's citizen facing, in which case there will be other routes in (mostly starting on GOV.UK).